cfme/utils/appliance/db.py

```diff
@@ -229,47 +227,6 @@ def setup(self, **kwargs):
 
         self.logger.info('DB setup complete')
 
-    def loosen_pgssl(self, with_ssl=False):
-        """Loosens postgres connections"""
-
-        self.logger.info('Loosening postgres permissions')
-
-        # Init SSH client
-        client = self.appliance.ssh_client
-
-        # set root password
-        cmd = "psql -d vmdb_production -c \"alter user {} with password '{}'\"".format(
-            conf.credentials['database']['username'], conf.credentials['database']['password']
-        )
-        client.run_command(cmd)
-
-        pg_prefix = self.pg_prefix
-        # back up pg_hba.conf
-        client.run_command(
-            'mv {pg_prefix}/var/lib/pgsql/data/pg_hba.conf '
-            '{pg_prefix}/var/lib/pgsql/data/pg_hba.conf.sav'.format(pg_prefix=pg_prefix))
-
-        if with_ssl:
-            ssl = 'hostssl all all all cert map=sslmap'
-        else:
-            ssl = ''
-
-        # rewrite pg_hba.conf
-        write_pg_hba = dedent("""\
-        cat > {pg_prefix}/var/lib/pgsql/data/pg_hba.conf <<EOF
-        local all postgres,root trust
-        host all all 0.0.0.0/0 md5
```

Comment:
Per your PR comment about the default configuration, not including this line will prevent remote hosts from being able to connect over non-SSL connections.   Does the DB client in the framework still work without this? Is it using ssl for its connections?

Reply:
It seems it is not needed. I explained this the comment to this PR.